.github/workflows/e2e-fixture-test.yml

```diff
@@ -0,0 +1,38 @@
+name: E2E Fixture Test
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - '**/*.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'hack/*.sh'
+      - 'LICENSE'
+      - 'SECURITY_CONTACTS'
+      - 'DCO'
+      - 'OWNERS'
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+    - name: Install Dependencies
+      run: "${GITHUB_WORKSPACE}/hack/e2e/ensure_cmctl.sh"
```

Comment:
With https://github.com/metal3-io/baremetal-operator/pull/1440 merged, we don't need this anymore.

Reply:
this aswell?
    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
    ??

Reply:
Checkout action checks out repo, so it is needed.

Reply:
No no. I meant only the `ensure_cmctl.sh` part (the script doesn't exist anymore anyway). Clicked on line 27 but don't know why it included the other 3 lines, sorry :sweat_smile:

Reply:
Ah good point @mquhuy ! Yes we should remove the call to `ensure_cmctl.sh`